docs/website/docs/deployment-configurations/cpu-local.md

```diff
@@ -121,7 +126,7 @@ In the build directory, run the following command:
 ``` shell hl_lines="2"
 tools/iree-run-module \
     --driver=local-task \
-    --module_file=mobilenet-dylib.vmfb \
+    --module_file=mobilenet-local-cpu.vmfb \
```

Comment:
the vmfbs themselves don't care about how they are executed, so this would just be `-cpu` (or `-[arch]` like `-vmvx` or `-x86_64`) - that's because it'll run with local-sync/local-task/remote-*/sandbox-/local-taskflow/whatever and the only thing that matters is what kind of executables are in it.